ui/admin/tests/acceptance/workers/update-test.js

```diff
@@ -0,0 +1,98 @@
+import { module, test } from 'qunit';
+import { visit, click, find, fillIn } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { Response } from 'miragejs';
+import {
+  authenticateSession,
+  // These are left here intentionally for future reference.
+  //currentSession,
+  //invalidateSession,
+} from 'ember-simple-auth/test-support';
+
+module('Acceptance | workers | update', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  const instances = {
+    scopes: {
+      global: null,
+    },
+  };
+
+  const urls = {
+    globalScope: null,
+    workers: null,
+    worker: null,
+  };
+
+  hooks.beforeEach(function () {
+    //Generate the resources
+    instances.scopes.global = this.server.create('scope', { id: 'global' });
+    instances.worker = this.server.create('worker', {
+      scope: instances.scopes.global,
+    });
+    // Generate route URLs for resources
+    urls.globalScope = '/scopes/global';
+    urls.workers = `${urls.globalScope}/workers`;
+    (urls.worker = `${urls.workers}/${instances.worker.id}`),
+      authenticateSession({});
+  });
+
+  test('can save changes to an existing worker', async function (assert) {
+    assert.expect(2);
+    await visit(urls.worker);
+    await click('form [type="button"]', 'Click edit mode');
+    await fillIn('[name="name"]', 'Updated worker name');
+    await click('.rose-form-actions [type="submit"]');
+    assert.dom(`[href="${urls.worker}"]`).isVisible();
+    await assert.dom('input[name="name"]').hasValue('Updated worker name');
+  });
+
+  test('can cancel changes to an existing worker', async function (assert) {
+    assert.expect(1);
+    let name = instances.worker.name;
```

Comment:
```suggestion
    const name = 'Updated Worker Name';
```

I think it was fine to still use the `Updated Worker Name` as you need a way to differentiate between what was already there. Right now you're updating the name with the same name as it was before so we can't tell if it changed or not

Reply:
Yea didn't mean for the `${name}` to be entered into the `fillIn` helper, but the idea of the test is to make sure that current value of the input field isn't changed after we click the cancel button correct?.

Reply:
Right the fill in can just be the throwaway text, I thought you were still checking with `doesNotContainText`, just change the fill in that case